### DISCUSSION TITLE
How do I specify alignment in react-aria useTable?

### DISCUSSION THREAD

**Discussion by kristiandupont** [adobe/react-spectrum#2718] [Q&A]
I have built a table component using `useTable` and I would like the consumer to be able to specify how to align each column (left, center, right). I can see that the Spectrum `Column` component allows this and as far as I can tell, it works by [casting](https://github.com/adobe/react-spectrum/blob/main/packages/%40react-spectrum/table/src/index.ts#L20) the Column type? 

I can do this as well, of course, but it seems like a hack to a very common thing in tables so if that is indeed the way to go, I am curious as to why this property wasn't added to Aria?

**Comment by reidbarber** [comment]
The main reason it isn't included is because React Aria intentionally doesn't implement any styles by itself. I think React Spectrum's method you mentioned is probably the best bet for achieving that functionality.
